docs/modules/qbi-helper/interest-query.md

```diff
@@ -0,0 +1,53 @@
+<%(TOC)%>
+
+# How to construct an InterestQuery
+
+<%(Callout message="
+Before reading this document, make sure you have read:
+
+  * [Introduction to Query-based interest]({{urlRoot}}/modules/qbi-helper/intro-to-qbi)
+  * [How to use InterestTemplate]({{urlRoot}}/modules/qbi-helper/interest-template)
+")%>
+
+An [`InterestQuery`]({{urlRoot}}/api/query-based-interest/interest-query) object contains the constraint of a query and the components that the query will return from entities that match the given constraint.
+
+> In the future, the Runtime will also provide functionality to limit the frequency at which a query returns results.
+
+## Create a query
+
+To create an interest query, call `InterestQuery.Query` and provide a [`Constraint`]({{urlRoot}}/api/query-based-interest/constraint). For example:
+
+```csharp
+var query = InterestQuery
+    .Query(Constraint.RelativeSphere(20));
```

Comment:
does that need to be on 2 lines?

Reply:
I was trying to be consistent with how we put `.Query` on a separate line in all other code samples